core/runtime/src/main/java/io/quarkiverse/jberet/runtime/JBeretRecorder.java

```diff
@@ -57,12 +58,12 @@ public void registerJobs(List<Job> jobs) {
         return new RuntimeValue<>(new JBeretConfigSourceProvider(properties));
     }
 
-    public void initJobOperator(final JBeretConfig config, final BeanContainer beanContainer) {
+    public void initJobOperator(final JBeretConfig config, final BeanContainer beanContainer, final ThreadPoolConfig threadPoolConfig) {
```

Comment:
Can you group both config classes? So, arguments order should be first the config objects and then the `BeanContainer`.

Reply:
@radcortez , Order of config objects is done. Is your idea to group both config files to one file? But as Config root for both config files are different it is best not to merge. If your idea is different then that, please guide me through what approach you have.

Reply:
Thanks. It was just the order :)